ocl/test_helper/base.py

```diff
@@ -240,3 +241,32 @@ def create_lookup_concept_classes(user, org_ocl):
     create_concept(user, locales_source, concept_class="Locale", names=[create_localized_text("tr")])
     create_concept(user, locales_source, concept_class="Locale", names=[create_localized_text("Abkhazian", "en")])
     create_concept(user, locales_source, concept_class="Locale",names=[create_localized_text("English", "en")])
+
+def create_dictionary(user, validation_schema=None, organization=None, name=None):
+    suffix = generate_random_string()
+
+    dictionary = Dictionary(
+        name=name if name else "dictionary{0}".format(suffix),
+        mnemonic=name if name else "dictionary{0}".format(suffix),
+        full_name=name if name else "Dictionary {0}".format(suffix),
+        public_access=ACCESS_TYPE_EDIT,
+        default_locale='en',
+        preferred_source='CIEL',
+        supported_locales=['en'],
+        website='www.dictionary2.com',
+        description='This is the first  test dictionary',
+        custom_validation_schema=validation_schema
+    )
+
+    if organization is not None:
+        kwargs = {
+            'parent_resource': organization
+        }
+    else:
+        kwargs = {
+            'parent_resource': UserProfile.objects.get(user=user)
+        }
+
+    Dictionary.persist_new(dictionary, user, **kwargs)
+
+    return Dictionary.objects.get(id=dictionary.id)
```

Comment:
Could you add a new line at the end of the file

Reply:
resolved

Reply:
I have resolved this @shakirandagire